ADO.NET/SampleApp.cs

```diff
@@ -2,15 +2,19 @@
 using System.Configuration;
 using Vertica.Data.VerticaClient;
 
-class SampleApp
+internal class SampleApp
 {
     static void Main(string[] args)
     {
         // Create a connection
         using (VerticaConnection connection = new VerticaConnection())
         {
             connection.ConnectionString = ConfigurationManager.ConnectionStrings["ConnectionString"].ConnectionString;
-            
+            Console.WriteLine ("Vertica ADO.net sample application.");

```

Comment:
Nit: ADO.NET should always be capitalized.

Reply:
Done